stable/graphite/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
-version: 0.1.4
```

Comment:
The changes should justify at least a minor version bump.
So 0.2.0 would fit better.

Reply:
I'll bump it to 0.2.0